### PR TITLE
Implement ISR for all Raspberry Pi Boards - new driver interface

### LINKED ISSUE #204: WiringPi ISR does not work with current Kernel (6.6+)
By dropping GPIO sysfs WiringPi ISR does not work anymore.

**users currently relying on WiringPi should _NOT_ upgrade to the latest kernel at the moment, please stand by for another week or so**

This is a known bug and the maintainer team is working on quickfix that'll re-enable basic ISR support as well as a long-term solution that'll ensure all hardware revisions are supported in a meaningful manner.

More information may be found in discussions:
* https://github.com/WiringPi/WiringPi/discussions/201
* https://github.com/WiringPi/WiringPi/discussions/202

### PR DESCRIPTION
**Pull request by mstroh76**
With these changes, wiringPi utilizes the same GPIO ABI as libgpiod.

fixes #204
implements #200 
implements #208

### CODE REVIEW

File: reinstall
Comment (mwallner) [inline reinstall:1]: I guess this file got here by accident?

File: version.h
Comment (mwallner) [inline version.h:1]: for the next time, it'd be great if we could have a separate commit for bumping the version (just before the actual release) - this way people who may build from source / a non-master branch wouldn't get the "new version" in advance. (it may actually not be the final version - therefore we should only bump the version at the very last moment possible)

**Review by mwallner — APPROVED** [review]
generally speaking these changes look good, does this mean we'll have ISR working _again_ on Raspberry Pi <= 4 with the latest kernel versions AS WELL AS support for Pi5? :exploding_head:

**Comment by mstroh76** [comment]
Yes

Manfred Wallner ***@***.***> schrieb am Fr. 22. März 2024 um
20:19:

**Comment by mstroh76** [comment]
New ISR works with all Pi's and old and new kernel as well.

There is even the possibility to stop isr , that was hardly missing before.
